cdap-ui/app/cdap/components/Experiments/FieldLevelLineage/Header/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import { Consumer } from '../FllContext';
+
+interface IHeaderProps {
+  type: string;
+  first: number;
+  total: number;
+}
+
+function Header({ type, first, total }: IHeaderProps) {
+  return (
+    <Consumer>
+      {({ numTables, target }) => {
+        let last;
+        if (type === ('impact' || 'cause')) {
+          last = first + numTables - 1 <= total ? first + numTables - 1 : total;
+        } else {
+          last = total;
+        }
+
+        const header =
+          type === 'target'
+            ? 'Select a field to view lineage and operations'
```

Comment:
I would suggest start `i18n` right from the beginning. Will ease out our work at the end when the feature is complete but not the `i18n` for FLL